code/nathan/python/credit_validation.py

```diff
@@ -1,4 +1,4 @@
-def is_credit_number_valid(credit_number): #expecting a string as an argument
+def is_credit_number_valid(credit_number):
     credit_number = list(credit_number)
     check_digit = credit_number.pop(-1)
```

Comment:
Food for thought. The .pop() method with no arguments passed in the '.pop(*arg)' removes the last variable from the list by default. So the -1 isn't required here but it doesn't break anything.